monkeys/src/main/kotlin/com/wire/kalium/monkeys/MonkeyApplication.kt

```diff
@@ -70,8 +73,12 @@ class MonkeyApplication : CliktCommand(allowMultipleSubcommands = true) {
 
         coreLogic.updateApiVersionsScheduler.scheduleImmediateApiVersionUpdate()
         val testData = TestDataImporter.importFromFile(dataFilePath)
+        // TODO: when getAllKnownContacts is fixed to return contacts for the caller's team, remove this
+        // https://wearezeta.atlassian.net/browse/WPB-3261
+        val userIdsPerBackend =
```

Comment:
Fixed already. PR to be merged as soon as we fix some broken stuff:

https://github.com/wireapp/kalium/pull/1949

Reply:
I'll wait this to be merged and remove this

Reply:
Merged:

- https://github.com/wireapp/kalium/pull/1949